css/style.css

```diff
@@ -902,6 +910,12 @@ button.button-inline:hover {
 	}
 }
 
+@media (max-width: 768px) {
+	#app-navigation-toggle {
+		display: none !important;
```

Comment:
I think we could dispense with navigation bar completely if we move all buttons/controls into board view.